src/main/groovy/Index.groovy

```diff
@@ -25,7 +25,7 @@ class Index implements RestApiController {
         try {
             def rows = params.isEmpty() ? sql.rows(query) : sql.rows(query, params)
             JsonBuilder builder = new JsonBuilder(rows)
-            String table = builder.toPrettyString()
+            String table = builder.toString()
```

Comment:
do you have an example of failing complex data that doesn't work with toPrettyString() ?